lib/synthesize-stream.ts

```diff
@@ -126,17 +126,42 @@ class SynthesizeStream extends Readable {
 
     socket.onmessage = message => {
       const chunk = message.data;
-      // some messages are strings - emit those unencoded, but push them to
-      // the stream as binary
-      const data = typeof chunk === 'string' ? chunk : Buffer.from(chunk);
+      // some info messages are sent as strings, telling the content_type and
+      // timings. Emit them as separate events, but do not send them along the
+      // pipe.
+      if (typeof chunk === 'string') {
+        try {
+          const json = JSON.parse(chunk);
```

Comment:
technically, I think if it's a string, it should always be a JSON object so the try/finally might not be needed, which would allow for some simplification of this logic (and the two lines following it). Let me know how defensive you think this should be.

Reply:
I also figured it was more appropriate to try and separate things out as much as possible instead of just outputting all strings to a user and letting them figure things out, but can condense if desired.

Reply:
I think it's good to be this defensive here. It's JavaScript after all, plus we don't have control over what comes back from the service. Nice to know this won't choke on anything weird

Reply:
👍